test/py/test_pci_caps.py

```diff
@@ -293,9 +294,29 @@ def test_pci_cap_write_pmcs():
 
     disconnect_client(ctx, sock)
 
+@c.CFUNCTYPE(c.c_int, c.c_void_p, c.c_int)
+def vfu_reset_cb(ctx, reset_type):
+    assert reset_type == VFU_RESET_PCI_FLR or reset_type == VFU_RESET_LOST_CONN
+    return 0
+
 def test_pci_cap_write_px():
-    # FIXME
-    pass
+    sock = connect_client(ctx)
+    ret = vfu_setup_device_reset_cb(ctx, vfu_reset_cb)
+    assert ret == 0
+
+    #flrc
+    cap = struct.pack("ccHHcc52c", to_byte(PCI_CAP_ID_EXP), b'\0', 0, 0, b'\0',
+                      b'\x02', *[b'\0' for _ in range(52)])
```

Comment:
doesn't just
```
b'\0' * 52
```

work ?

Reply:
It doesn't work,  

>>> pack("52c", b'\0' * 52)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
struct.error: pack expected 52 items for packing (got 1)

Reply:
right, makes sense.